provider-directory/src/test/java/gov/va/api/health/providerdirectory/service/controller/practitioner/PractitionerControllerTest.java

```diff
@@ -51,7 +52,7 @@ public void searchByFamilyAndGiven() {
             .readValue(
                 getClass().getResourceAsStream("/test-search-by-family-and-given.json"),
                 Practitioner.Bundle.class);
-    assertThat(actual).isEqualTo(expected);
+    Assertions.assertThat(actual).isEqualTo(expected);
```

Comment:
assertThat is already statically imported elsewhere in the file, so you can revert this change